pkgs/applications/editors/vscode/vscode.nix

```diff
@@ -46,6 +46,8 @@ in
 
     updateScript = ./update-vscode.sh;
 
+    dontFixup = stdenv.isDarwin;
```

Comment:
A short comment why we want this wouldn't hurt

Reply:
Sure, I'll just use the commit message if that copy looks good.